### PR TITLE
praat: update to 6.0.40

### PR DESCRIPTION
**Pull request by newbluemoon**

### CODE REVIEW

File: srcpkgs/praat/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-version=6.0.38
+version=6.0.40
 revision=1
 hostmakedepends="pkg-config"
-makedepends="gtk+-devel alsa-lib-devel $(vopt_if pulseaudio pulseaudio-devel)"
+makedepends="alsa-lib-devel gtk+-devel $(vopt_if pulseaudio pulseaudio-devel)"
 short_desc="Speech analysis, synthesis, and manipulation"
 maintainer="newbluemoon <blaumolch@mailbox.org>"
 license="GPL-2.0-or-later"
 homepage="http://www.praat.org/"
```
Comment (maxice8) [inline srcpkgs/praat/template]: this url redirects to http://www.fon.hum.uva.nl/praat/

Comment (newbluemoon) [reply srcpkgs/praat/template]: Is this a problem? On the bottom of http://www.fon.hum.uva.nl/praat/ it says
> This page is www.praat.org.

So I assume www.praat.org is the real homepage and the redirection could change at any time.

**Comment by maxice8** [comment]
On Sat, May 12, 2018 at 12:49:51AM -0700, newbluemoon wrote:
> @newbluemoon commented on this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In srcpkgs/praat/template:
> 
> >  revision=1
>  hostmakedepends="pkg-config"
> -makedepends="gtk+-devel alsa-lib-devel $(vopt_if pulseaudio pulseaudio-devel)"
> +makedepends="alsa-lib-devel gtk+-devel $(vopt_if pulseaudio pulseaudio-devel)"
>  short_desc="Speech analysis, synthesis, and manipulation"
>  maintainer="newbluemoon <blaumolch@mailbox.org>"
>  license="GPL-2.0-or-later"
>  homepage="http://www.praat.org/"
> 
> Is this a problem? On the bottom of http://www.fon.hum.uva.nl/praat/ it says
> 
>     This page is www.praat.org.
> 
> So I assume www.praat.org is the real homepage and the redirection could change
> at any time.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

oh yeah i didn't notice that, my bad.

**Comment by newbluemoon** [comment]
No problem! :)

**Pull request closed without merging**